holoviews/core/data/dask.py

```diff
@@ -229,12 +229,8 @@ def dframe(cls, columns, dimensions):
         return columns.data.compute()
 
     @classmethod
-    def length(cls, dataset):
-        """
-        Length of dask dataframe is unknown, always return 1
-        for performance, use shape to compute dataframe shape.
-        """
-        return 1
+    def nonzero(cls, dataset):
+        return True
 
```

Comment:
I tried various things such as using:

```python
try:
   next(df.iterrows())
except:
   nonzero = False
else:
   nonzero = True
```

But all of these approaches still take a considerable amount of time.